sp800-63b/sec6_lifecycle.md

```diff
@@ -8,6 +8,8 @@ During the lifecycle of an authenticator bound to a subscriber's identity, a num
 
 ### <a name="binding"></a>6.1. Authenticator Binding
 
+*Authenticator binding* refers to an association that is established providing the ability for a given authenticator to be used, possibly in conjunction with others, to authenticate to a given subscriber account.
```

Comment:
It's really the association with the account; proofing establishes the association of the account with the rightful subscriber. But otherwise OK.